extensions/wikia/Recirculation/templates/Recirculation_discussions.php

```diff
@@ -0,0 +1,33 @@
+<section class="mcf-card mcf-card-discussions">
+	<header class="mcf-card-discussions__header">
+		<span>Latest Discussions</span>
+		<a href="<?= $discussionsUrl ?>" class="mcf-card-discussions__link">View All
+			<?= DesignSystemHelper::renderSvg(
+				'wds-icons-arrow',
+				'wds-icon wds-icon-small mcf-card-discussions__link-icon'
+			) ?></a>
+	</header>
+	<ul>
+		<?php foreach ( $posts as $post ): ?>
+			<li class="mcf-card-discussions__item">
+				<a href="/User:<?= $post->author ?>" class="mcf-card-discussions__user-info">
```

Comment:
Namespace should be internationalized. You can use `Title::newFromText($post->author, NS_USER)->getFullURL()`